src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -216,7 +217,14 @@ public void Dispose()
         #region ILogSource Implementation
         private void Log(string message)
         {
-            viewModel.MessageLogged?.Invoke(LogMessage.Info(message));
+            if (DynamoModel.IsTestMode)
+            {
+                System.Console.WriteLine(message);
```

Comment:
does this actually make it into the test log?

Reply:
YEs, it does